app/views/course/achievements/_form.html.slim

```diff
@@ -5,6 +5,10 @@
 
   = f.input :description
 
+  = f.label t('.badge')
+  div.achievement-form = display_achievement_badge(@achievement)
+  = f.file_field :badge, class: 'upload-btn'
```

Comment:
is there a need to style the file field in the first place? Or is it something all file fields should be styled with?